pkg/apis/kops/validation/legacy.go

```diff
@@ -434,6 +434,12 @@ func ValidateCluster(c *kops.Cluster, strict bool) *field.Error {
 			}
 		}
 
+		if c.Spec.Kubelet.BootstrapKubeconfig != "" {
+			if c.Spec.KubeAPIServer == nil {
```

Comment:
I'm guessing you're going to add this one in a follow-on PR (sequencing issues?)